pkg/config/config.go

```diff
@@ -107,6 +108,10 @@ type TRPIdentityConfig struct {
 	LEI      string `required:"false" desc:"the lei of your vasp to respond to a trp identity request"`
 }
 
+type SunriseConfig struct {
+	Enabled bool `default:"true" desc:"if false, it will not be possible to send sunrise emails and sunrise endpoints will return a 404"`
```

Comment:
Setting the value to `true` by default here feels like the easiest way to ensure that each node has it enabled while turning it off is a simple change to the config. If we get feedback requesting another way to enable/disable the feature when we could reassess.